modules/system/assets/js/mailbrandsettings/mailbrandsettings.js

```diff
@@ -33,5 +33,7 @@ function createPreviewContainer(el, content) {
      * Auto adjust height
      */
     $(document).render(adjustPreviewHeight)
-    $(window).resize(adjustPreviewHeight)
+    $(window).on('resize', function() {
```

Comment:
@ayumihamsaki I think you could simplify.
```suggestion
    $(window).on('resize', adjustPreviewHeight);
```

Reply:
done